docs/Other SDL Features/Using App Services/index.md

```diff
@@ -126,10 +126,9 @@ SDLAppServiceRecord *serviceRecord = aCapability.updatedAppServiceRecord;
 let getResponse: SDLGetSystemCapabilityResponse = <#From wherever you got it#>
 
 // This array contains all currently available app services on the system
-guard let capabilities = getResponse.systemCapability?.appServicesCapabilities, let appServices : [SDLAppServiceCapability] = capabilities.appServices else {
+guard let capabilities = getResponse.systemCapability?.appServicesCapabilities, let appServices = capabilities.appServices, let aCapability = appServices.first else {
     return
 }
-let aCapability = appServices.first
 
 // This will be nil since it's the first update
 let capabilityReason = aCapability?.updateReason
```

Comment:
Now that `aCapability` is no longer an optional, you need to remove the `?`. Please put it into Xcode to test and make sure it all works again.